.github/workflows/ci-c-warnings.yml

```diff
@@ -53,6 +53,18 @@ jobs:
       - name: Enable nodiscard
         if: ${{ matrix.case == 0}}
         run: echo "CMAKE_AVIF_FLAGS=\"-DAVIF_ENABLE_NODISCARD=ON\""  >> $GITHUB_ENV
+      - name: Use stdbool for AVIF_TRUE/AVIF_FALSE
+        if: ${{ matrix.case == 1}}
+        run: |
+          sed -i 's/#include <stdint.h>/#include <stdint.h>\n#include <stdbool.h>/' include/avif/avif.h
+          sed -i 's/typedef int avifBool;/typedef bool avifBool;/' include/avif/avif.h
+          sed -i 's/#define AVIF_TRUE 1/#define AVIF_TRUE true/' include/avif/avif.h
+          sed -i 's/#define AVIF_FALSE 0/#define AVIF_FALSE false/' include/avif/avif.h
+      - name: Use enums for AVIF_TRUE/AVIF_FALSE
+        if: ${{ matrix.case == 2}}
+        run: |
+          sed -i 's/#define AVIF_TRUE 1/enum avifTrueFalse_{ AVIF_TRUE = 1,/' include/avif/avif.h
+          sed -i 's/#define AVIF_FALSE 0/AVIF_FALSE = 0};/' include/avif/avif.h
```

Comment:
I suggest we add these two alternative definitions of `avifBool` to avif.h and avoid patching avif.h like this. The alternative definitions would be guarded by macros that we define in this CI workflow.

If we want to patch avif.h, we may want to add comments around the definition of `avifBool` to prevent people form changing or even reformatting it.

Reply:
I think adding two guards (for stdbool and enum) is a lot of noise for a public header no? Especially just for a CI usage (not even a test).